.travis.yml

```diff
@@ -15,10 +15,12 @@ before_install:
     - sudo pip install pyopenssl

```

Comment:
@delta24 Carriage return were not introduced in this PR. They are in develop branch.